### PR TITLE
Add /bbl route

### PR DESCRIPTION
**Pull request by pichot**
Simple commit that adds a `/bbl` route. Uses already implemented `bbl-demux` to parse bbl and redirect to `/lot` route.

Unit and acceptance tests included.

### CODE REVIEW

File: app/routes/bbl.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import bblDemux from '../utils/bbl-demux';
+
+export default Ember.Route.extend({
+  model(params) {
```
Comment (pichot) [inline app/routes/bbl.js]: @allthesignals is this the right hook to use? I tried `beforeModel` but it appeared that the params weren't fully loaded.

**Comment by pichot** [comment]
Rebased and ready to merge please :)

File: app/routes/bbl.js (hunk 1 above)
Comment (allthesignals) [reply app/routes/bbl.js]: @pichot yes - beforeModel doesn't have the params object directly, you'd have to access it differently.

File: app/routes/bbl.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    const bbl = bblDemux(params.bbl);
```
Comment (allthesignals) [inline app/routes/bbl.js]: You might also have said:

```javascript
export default Ember.Route.extend({
  model({ bbl }) {
    const { boro, block, lot } = bblDemux(bbl);
    this.transitionTo('lot', boro, block, lot);
  }
});
```

Comment (pichot) [reply app/routes/bbl.js]: oooh, better. Should have waited for your code review

**Pull request merged**